test/unit/selector.js

```diff
@@ -2125,7 +2125,7 @@ QUnit.test( "jQuery.escapeSelector", function( assert ) {
 	assert.equal( jQuery.escapeSelector( "\x20\x21\x78\x79" ), "\\ \\!xy",
 		"Escapes non-word ASCII characters" );
 
-	// Astral symbol (U+1D306 TETRAGRAM FOR CENTRE)
```

Comment:
This is the official name for the character so it shouldn't be changed: https://www.fileformat.info/info/unicode/char/1d306/index.htm